bazel-java-deps.bzl

```diff
@@ -104,8 +105,9 @@ def install_java_deps():
             "org.junit.jupiter:junit-jupiter-engine:5.0.0",
             "org.junit.platform:junit-platform-engine:1.0.0",
             "org.junit.platform:junit-platform-runner:1.0.0",
-            "org.mockito:mockito-core:3.2.0",
-            "org.mockito:mockito-inline:3.2.0",
+            "org.mockito:mockito-core:2.24.0",
```

Comment:
I downgraded the Mockito libraries as this is the last to be compatible with the version of `scalatest`/`scalactic` we are using (3.0.5). I chose the version of `mockito-scala` so that it's compatible with both.

For as much as I like being on the latest version of a library, this seems to work and allows to now have possible weird runtime behaviors due to incompatible libraries.

https://mvnrepository.com/artifact/org.mockito/mockito-core/2.24.0
https://mvnrepository.com/artifact/org.mockito/mockito-scala_2.12/1.1.2

Reply:
Correction: I chose a version of `mockito-scala` that is quite old but it's the latest **that defines as its dependency** the version of `scalatest`/`scalactic` we are using (3.0.5).